packages/ui/components/pages/Fuse/FusePoolPage/MarketsList/FundButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Button, useDisclosure } from '@chakra-ui/react';
+import { FundOperationMode } from '@midas-capital/types';
+import { useMemo } from 'react';
+
+import PoolModal from '@ui/components/pages/Fuse/Modals/PoolModal/index';
+import { FUNDOPERATION_MODE_NAMES } from '@ui/constants/index';
+import { MarketData } from '@ui/types/TokensDataMap';
+
+export const FundButton = ({
+  comptrollerAddress,
+  assets,
+  asset,
+  mode,
+  isDisabled,
+}: {
+  comptrollerAddress: string;
+  assets: MarketData[];
+  asset: MarketData;
+  mode: FundOperationMode;
+  isDisabled?: boolean;
+}) => {
+  const { isOpen: isModalOpen, onOpen: openModal, onClose: closeModal } = useDisclosure();
+
+  const modeName = useMemo(() => FUNDOPERATION_MODE_NAMES[mode], [mode]);
```

Comment:
We could potentially get the name of the FundOperation from the Enum itself. We can assign strings to Enums which we could than retrieve, so no need for externally keeping the `FUNDOPERATION_MODE_NAMES`.

https://bobbyhadz.com/blog/typescript-convert-enum-to-string

Reply:
Fixed